INTERNALS.md

```diff
@@ -57,27 +57,27 @@ All Redis keys are namespaced, under `rm:` by default.
 
   The set of all subscriber UUIDs.
 
-`topics:{uuid}`
+`topics:{subscriber.name}`
```

Comment:
The subscriber "name" is identical to its "client token" (formerly UUID), and normally different from its "delivery token" (formerly UUID as well). So I'd recommend using consistent naming here.